lib/puppet/provider/cs_clone/pcs.rb

```diff
@@ -28,7 +28,7 @@ def change_clone_id(type, primitive, id, cib)
     return unless doc.root.attributes['id'] != id
 
     doc.root.attributes['id'] = id
-    cmd = [command(:cibadmin), '--replace', '--xpath', xpath, '--xml-text', doc.to_s.chop]
+    cmd = [command(:cibadmin), '--replace', '--xpath', xpath, '--xml-text', doc.to_s]
```

Comment:
which errors did that cause? Maybe this was meant to be chomp. not chop?

Reply:
i agree chomp is the best solution! i it was cutting in half a new line.

Reply:
oh you did find that also. wasn't aware of it